cpp/arcticdb/version/version_store_api.hpp

```diff
@@ -182,11 +182,11 @@ class PythonVersionStore : public LocalVersionedEngine {
         const VersionQuery& version_query
     );
 
-    std::vector<std::pair<VersionedItem, py::object>> batch_read_descriptor(
+    std::vector<std::pair<DescriptorItem, py::object>> batch_read_descriptor(
         const std::vector<StreamId>& stream_ids,
         const std::vector<VersionQuery>& version_queries);
 
-    std::pair<VersionedItem, py::object> read_descriptor(
+    std::pair<DescriptorItem, py::object> read_descriptor(
```

Comment:
Both of these APIs would be cleaner if `DescriptorItem` had the time series descriptor protobuf as a member, with a method in python_bindings.cpp to convert it to a `py::object`